gradle/libs.versions.toml

```diff
@@ -110,6 +111,9 @@ test_mockk = { module = "io.mockk:mockk", version.ref = "test_mockk" }
 test_barista = { module = "com.adevinta.android:barista", version.ref = "test_barista" }
 test_hamcrest = { module = "org.hamcrest:hamcrest", version.ref = "test_hamcrest" }
 test_orchestrator = { module = "androidx.test:orchestrator", version.ref = "test_orchestrator" }
+test_turbine = { module = "app.cash.turbine:turbine", version.ref = "test_turbine"}
+test_truth = "com.google.truth:truth:1.1.3"
```

Comment:
Not the same format than others?

Reply:
Yes, I think we should use this format when there is no shared version between libs. What do you think?

Reply:
This is definitely more readable 👍 
Also if possible we should prioritize usage of `bom`.